packages/google_maps_flutter/example/test_driver/google_maps_e2e.dart

```diff
@@ -307,15 +302,15 @@ void main() {
     expect(scrollGesturesEnabled, true);
   });
 
-  test('testGetVisibleRegion', () async {
+  testWidgets('testGetVisibleRegion', (WidgetTester tester) async {
```

Comment:
@iskakaushik Could you please review the changes in this test. Since there are some tiny modifications of the original test.

Reply:
LGTM